creating/creating_index.adoc

```diff
@@ -482,6 +483,73 @@ And then we need to change the CMD instruction to call _/usr/sbin/init_ to let s
 RUN /usr/sbin/init
 ```
 
+==== Using systemd to control containers
+
+The control mechanism for most docker functions is done via the docker commands or something like
+the atomic application which simplifies the management of containers and images for users.  But in
+a non-development environment, you may wish to treat your containers more like traditional services
+or applications.  For example, you may wish to have your containers start in a specific order on
+boot-up.  Or perhaps you wish to be able to restart (or recycle) a container because you have changed
+its configuration file.
+
+There are several approaches to these sorts of function.  You can make sure a specific container
+always starts on boot-up using the --restart switch with the docker command line when you initially
+run the image.  There are also helper applications like Kubernetes that will allow you to determine the
+start up order of containers even when they are distributed.  But in the case where all the containers
+reside on a single node, systemd might just be exactly the solution.  Like with traditional services,
+system is capable of making sure services both start and in the order they are specified.  Moreover,
+any issues with startup or the container are logged like any other system service.
+
```

Comment:
Let's add a pros/cons list here (or further down).

**When to use Systemd to manage containers**

* For host system services such as agents and long-running services
* Logging via Journald is desired
* Traditional service management via `systemctl` is desired
* Service dependency management is required